src/models/common/decorators/fieldConfig/classFieldConfig.decorator.ts

```diff
@@ -0,0 +1,51 @@
+import 'reflect-metadata';
+import { getFieldConfig, IPropFieldConfigInfo } from './fieldConfig.decorator';
+
+const fieldConfigMetadataKey = Symbol('fieldconfigclass');
+type KeyValueDict = Record<string, unknown>;
+const target = {};
```

Comment:
I should have thought about it earlier but since target is static and global it can be used to store the data directly without using reflect metadata.
it will make the code bit simpler to understand.

up to you if u want to change it, its ok either way by my standard (for this use case)

Reply:
Done, reflection is removed, more clean code